pallets/bootstrapper.go

```diff
@@ -0,0 +1,47 @@
+package pallets
```

Comment:
Are you planning on extending this to include all the pallets we use, like `anchors` etc

Reply:
Yea, that would be the end goal.